src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,72 @@
 'use strict';
 
+const NAME = '';
```

Comment:
The constant `NAME` is defined but never used. Consider removing it if it's not going to be used in the future, to keep the code clean.